meson.build

```diff
@@ -0,0 +1,253 @@
+project('pysimbi', 'c', 'cpp', 'cython',
+    version : run_command('python', 'pysimbi/version.py', check: true).stdout().strip(), 
+    license : 'MIT',
+    default_options : [ 'cpp_std=c++17' ],
+)
+
+# BUILD CONFIG
+conf_data = configuration_data()
+conf_data.set('version', meson.project_version())
+conf_data.set10('FLOAT_PRECISION', get_option('float_precision'))
+configure_file(
+    input:  'build_options.hpp.in',
+    output: 'build_options.hpp',
+    configuration: conf_data
+)
+
+# DEPENDENCIES
+fs      = import('fs')
+pymod   = import('python')
+py3     = pymod.find_installation('python3', required: true)
+py3_dep = py3.dependency()
+omp     = dependency('openmp', language: 'cpp')
+hdf5    = dependency('hdf5', language: 'cpp', required: true)
+# ensure the install directory recognizes conda environment
+py3_purelib   = py3.get_path('purelib')
+py3env_prefix = py3.get_variable('prefix', 'empty')
+message('python path -- ' + py3.path())
+message('python install_dir -- ' + py3env_prefix)
+if py3_purelib.endswith('site-packages')
+    py3_install_dir = py3_purelib
+elif py3env_prefix.endswith('site-packages')
+    py3_install_dir = py3env_prefix
+else
+    py3_install_dir = py3.get_installation_dir()
+endif
```

Comment:
I'm not entirely clear on why this should be needed. Is there some case where `py3.get_install_dir()` or relying on the default install dir for `py3.extension_module()` isn't working? What does it produce instead?

Is this solved by configuring Meson with the same --prefix as conda itself, or using `-Dpython.install_env=auto`?